src/components/organisms/PaymentModal/PaymentForm.tsx

```diff
@@ -1,215 +0,0 @@
-import React, { useState, useEffect } from "react";
-import { useStripe, useElements, CardElement } from "@stripe/react-stripe-js";
-
-import { VenueEvent } from "types/venues";
-
-import CardInput from "components/molecules/CardInput";
-import firebase from "firebase/app";
-import "firebase/functions";
-import TabNavigation from "components/molecules/TabNavigation";
-import { PAYMENT_FORM_TAB_ARRAY, INDIVIDUAL_TICKET_TAB } from "./constants";
-import { useUser } from "hooks/useUser";
-import { useVenueId } from "hooks/useVenueId";
-import { Stripe, StripeElements } from "@stripe/stripe-js";
-import { WithId } from "utils/id";
-
-interface PropsType {
-  setIsPaymentSuccess: (value: boolean) => void;
-  setIsPaymentProceeding: (value: boolean) => void;
-  setIsCardBeingSaved: (value: boolean) => void;
-  isPaymentProceeding: boolean;
-  isCardBeingSaved: boolean;
```

Comment:
Have we made sure to clean up/remove these values from wherever used to call this component?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40